tests/test_io.py

```diff
@@ -12,31 +12,41 @@
 
 import pytest
 import zmq
-import zmq.asyncio
+import zmq_anyio
+from anyio import create_task_group
 from jupyter_client.session import Session
 
 from ipykernel.iostream import _PARENT, BackgroundSocket, IOPubThread, OutStream
 
+pytestmark = pytest.mark.anyio
```

Comment:
```suggestion
```

I think this is unused now.

Of if it's a module fixture, there should be a comment.

Reply:
I had to add it back in https://github.com/ipython/ipykernel/pull/1291/commits/192d62edd314199b15fba0bac766f3428c83724c.